only_otters/__main__.py

```diff
@@ -4,8 +4,8 @@
 
 
 if __name__ == '__main__':
-    app = QtWidgets.QApplication(sys.argv)
-    player = MusicPlayer()
-    player.show()
+    app = QApplication(sys.argv)
```

Comment:
`QApplication` is not resolved, you forgot to re-add the whole thing: `QtWidgets.QtApplication`.